### PR TITLE
Blessing

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: src/views/FSL/lookup/lookup.vue [hunk 1]
```diff
@@ -431,6 +430,7 @@
 													<td
 														class="text-capitalize"
 														v-for="repaymentBank in activeOrder.paymentBanks"
+														:key="repaymentBank.index"
```
Comment (teebarg) [inline src/views/FSL/lookup/lookup.vue:433]: does repaymentBank dict has key index?

File: src/views/FSL/payment_logger/payment_logger.vue [hunk 2]
```diff
@@ -9,7 +9,7 @@
 <script>
     import Lookup from "../lookup/lookup";
     import Flash from '../../../utilities/flash';
-    // import PaymentFilter from "../../../components/PaymentFilter";
+    // import PaymentFilter from "../../../components/PaymentFilter";
```
Comment (teebarg) [inline src/views/FSL/payment_logger/payment_logger.vue:12]: can we remove this if it's not used?

File: src/views/LOG/inventory/editComponent.vue [hunk 3]
```diff
@@ -176,6 +176,24 @@ export default {
   },
 
   methods: {
+    logTransfer(id, to) {
```
Comment (teebarg) [inline src/views/LOG/inventory/editComponent.vue:179]: what is this doing?

File: src/views/FSL/lookup/lookup.vue [hunk 4]
```diff
@@ -35,8 +35,7 @@
 						>
 							<div
 								class="mb-3 row attendance-item"
-								v-for="(order, index) in customer.orders"
-							>
+								v-for="(order, index) in customer.orders" :key="order.index"											>
```
Comment (teebarg) [inline src/views/FSL/lookup/lookup.vue:38]: is the index not unique? why not use it?

File: src/components/PaymentLog.vue [hunk 5]
```diff
@@ -348,6 +348,7 @@ export default {
       repaymentDuration: [],
       repaymentCyclesopt: [],
       downPaymentRates: [],
+      filteredDownPaymentRates:[],
```
Comment (teebarg) [inline src/components/PaymentLog.vue:351]: Are we still using this?

File: src/components/PaymentLog.vue [hunk 6]
```diff
@@ -309,7 +309,7 @@
 
                     <tr class="table-separator">
                       <th>Repayment Amount</th>
-                      <td v-for="am in amortization">
+                      <td v-for="{am} in amortization" :key="am.index">
```
Comment (teebarg) [inline src/components/PaymentLog.vue:312]: did you check there is an index key?

File: src/views/FSL/lookup/lookup.vue (hunk 1 above)
Comment (cheedinmar) [reply src/views/FSL/lookup/lookup.vue:433]: yes, every array has an index

File: src/views/FSL/payment_logger/payment_logger.vue (hunk 2 above)
Comment (cheedinmar) [reply src/views/FSL/payment_logger/payment_logger.vue:12]: ok, done

File: src/views/FSL/lookup/lookup.vue (hunk 1 above)
Comment (cheedinmar) [reply src/views/FSL/lookup/lookup.vue:433]: I have made changes on it

File: src/views/LOG/inventory/editComponent.vue (hunk 3 above)
Comment (cheedinmar) [reply src/views/LOG/inventory/editComponent.vue:179]: I did not make this change but the arguments are used in a post request to transfer products. {to: to_id,  id, inventory_id}

File: src/views/FSL/lookup/lookup.vue [hunk 7]
```diff
@@ -853,6 +860,7 @@
 				} else {
 					this.newOrder = false;
 					this.activeOrder = order;
+					console.log(this.activeOrder);
```
Comment (darmhoo) [inline src/views/FSL/lookup/lookup.vue:863]: remove log

**Review by darmhoo — APPROVED** [review]

**Review by teebarg — APPROVED** [review]

**Pull request merged**